src/styles/utils.scss

```diff
@@ -110,6 +110,10 @@
     cursor: pointer;
 }
 
+.smallButton {
```

Comment:
regular button gets its size from paddings (`padding: 12px 16px;`). Probably we want to use smaller paddings for small button

Reply:
done